config.js

```diff
@@ -53,10 +53,6 @@ module.exports = {
       'restify': path.join(__dirname, 'src/plugins/plugin-restify.js')
     },
 
-    // Valid entries are:
-    // 'express', 'hapi', 'http', 'restify'
-    excludedHooks: [],
```

Comment:
On that note -- how would users exclude plugins? Right now you'd have to obtain the value of `require('@google-cloud/trace/config.js').trace.plugins` and filter out the fields you don't want. Maybe we could expose this object somewhere else.